react.js

```diff
@@ -0,0 +1,207 @@
+/* eslint-env node */
```

Comment:
The same comment about the rules section and unnecessary project-related parameters
Take a look at ```angular.js```